src/rgw/rgw_request.h

```diff
@@ -59,9 +59,10 @@ struct RGWLoadGenRequest : public RGWRequest {
 	string method;
 	string resource;
 	int content_length;
-	std::atomic<int64_t>* fail_flag = nullptr;
+	std::atomic<bool>* fail_flag = { nullptr };
```

Comment:
@chardan This PR is for RBD-related stuff but this change is RGW. It would be easier to just open a new PR against RGW instead of having two different groups of reviewers on the same PR.